pkg/fqdn/cache.go

```diff
@@ -25,6 +25,8 @@ import (
 	"github.com/cilium/cilium/pkg/fqdn/matchpattern"
 	"github.com/cilium/cilium/pkg/ip"
 	"github.com/cilium/cilium/pkg/lock"
+	"github.com/cilium/cilium/pkg/logging/logfields"
+	"github.com/sirupsen/logrus"
```

Comment:
nit: I think cilium project is following the below import convention (i.e one extra line in between)

<std lib>

<cilium>

<thirdparty>

Reply:
Makes sense, fixed now.